project/src/test/kotlin/org/ionproject/core/utils/ControllerTester.kt

```diff
@@ -26,7 +26,7 @@ internal class ControllerTester {
         status { isOk }
         content { contentType("application/vnd.siren+json") }
         jsonPath("$.links") { exists() }
-        jsonPath("$.actions") { exists() }
+        //jsonPath("$.actions") { exists() }
```

Comment:
We need to figure a way to not make it mandatory but still check for certain actions when needed, like with `search` actions.